$NetBSD: patch-ad,v 1.3 2019/10/27 19:31:40 maya Exp $

Add dragonflybsd.
fix shell portability issues (= vs. ==)
switch xft-config to pkg-config.

--- configure.orig	2004-12-21 21:57:05.000000000 +0000
+++ configure
@@ -795,11 +795,15 @@ fi
 
 uname=`uname`
 
+if test "$uname" = "DragonFly"; then
+    LIBS="$LIBS -lkinfo"
+fi
+
 case $uname in
   Linux*)
     WANT_SYSINFO=yes
     ;;
-  FreeBSD*)
+  FreeBSD*|DragonFly*)
     WANT_KVM=yes
     ;;
   NetBSD*)
@@ -839,7 +843,7 @@ else
 fi
 
 
-if test x$uname = xFreeBSD; then
+if test x$uname = xFreeBSD || test x$uname = xDragonFly; then
   BUILD_FREEBSD_TRUE=
   BUILD_FREEBSD_FALSE='#'
 else
@@ -924,14 +928,14 @@ fi
 
 
 
-if test x$want_seti == xyes; then
+if test x$want_seti = xyes; then
   BUILD_SETI_TRUE=
   BUILD_SETI_FALSE='#'
 else
   BUILD_SETI_TRUE='#'
   BUILD_SETI_FALSE=
 fi
-if test x$want_seti == xyes; then
+if test x$want_seti = xyes; then
   cat >> confdefs.h <<\EOF
 #define SETI 1
 EOF
@@ -949,14 +953,14 @@ fi
 
 
 
-if test x$want_nvctrl == xyes; then
+if test x$want_nvctrl = xyes; then
   BUILD_NVCTRL_TRUE=
   BUILD_NVCTRL_FALSE='#'
 else
   BUILD_NVCTRL_TRUE='#'
   BUILD_NVCTRL_FALSE=
 fi
-if test x$want_nvctrl == xyes; then
+if test x$want_nvctrl = xyes; then
   cat >> confdefs.h <<\EOF
 #define NVCTRL 1
 EOF
@@ -1643,17 +1647,13 @@ else
   echo "$ac_t""no" 1>&6
 fi
 
-  if test x$XFTCONFIG != x; then
     
     cat >> confdefs.h <<\EOF
 #define XFT 1
 EOF
 
-    CFLAGS="$CFLAGS `$XFTCONFIG --cflags`"
-    XFT_LIBS=`$XFTCONFIG --libs`
-  else
-    want_xft=no
-  fi
+    CFLAGS="$CFLAGS `$PKG_CONFIG --cflags xft`"
+    XFT_LIBS=`$PKG_CONFIG --libs xft`
 fi
 
 
